### PR TITLE
Draft of measurePath

### PR DESCRIPTION
**Pull request by melchiar**
Began the draft of fabric.util.measurePath function for calculating the length of a pre-simplified path (using `makePathSimpler`)

TODO: still needs methods for C and Q

### CODE REVIEW

File: src/util/path.js [hunk 1]
```diff
@@ -395,6 +395,41 @@
...
+              y1 = current[2];
+            break;
+            case 'C':
+              //todo
+            break;
+            case 'Q':
+              //todo
+            break;
+        }
+    }
```
Comment (asturur) [inline src/util/path.js:430]: you probably have lint issue with those 4 spaces indent.

Comment (melchiar) [reply src/util/path.js:430]: hmm... i'll take a look

Comment (asturur) [reply src/util/path.js:430]: build is failing at lint step https://github.com/fabricjs/fabric.js/pull/6516/checks?check_run_id=977074861

Comment (melchiar) [reply src/util/path.js:430]: should be good now

**Comment by melchiar** [comment]
@asturur has something stalled the tests?

**Comment by asturur** [comment]
did not check, let me see.

**Comment by asturur** [comment]
github must have had some probelm with the checks, the build passed.

**Pull request merged**